website/content/en/status/report-2023-04-2023-06/mfsbsd.adoc

```diff
@@ -1,28 +1,31 @@
-=== Integrate mfsBSD into the release building tools 
+include::shared/authors.adoc[]
```

Comment:
I am unsure this is the right place to put that line. Maybe it would be more appropriate to put it in the _index.adoc file (which I am already preparing).

Reply:
Good point, I lost sight of the bigger picture. I'll fix after push.

Reply:
Fixed by https://github.com/freebsd/freebsd-doc/commit/25822abc88b404d30aa6ffecb5bdd160ea0d5b49